### PR TITLE
[id] Fixing inconsistency italic word for foreign word

### PR DESCRIPTION
**Pull request by rendiputra**
Fixing inconsistency italic word for foreign word in id/_index.html

### CODE REVIEW

File: content/id/_index.html [hunk 1]
```diff
@@ -8,13 +8,13 @@
 {{< blocks/section id="oceanNodes" >}}
 {{% blocks/feature image="flower" %}}
 ### Kubernetes (K8s)
-Kubernetes mengelompokkan kontainer yang membentuk suatu aplikasi dalam bentuk unit logis yang memudahkan proses manajemen dan <i>discovery</i>. Kubernetes dibuat berdasarkan [pengalaman operasional <i>workloads</i> skala <i>production</i> yang dilakukan oleh Google](http://queue.acm.org/detail.cfm?id=2898444), yang digabungkan dengan ide-ide terbaik dan best practices yang diberikan oleh komunitas.
+Kubernetes mengelompokkan kontainer yang membentuk suatu aplikasi dalam bentuk unit logis yang memudahkan proses manajemen dan <i>discovery</i>. Kubernetes dibuat berdasarkan [pengalaman operasional <i>workloads</i> skala <i>production</i> yang dilakukan oleh Google](http://queue.acm.org/detail.cfm?id=2898444), yang digabungkan dengan ide-ide terbaik dan <i>best practices</i> yang diberikan oleh komunitas.
```
Comment (ariscahyadi) [inline content/id/_index.html:11]: "production" mungkin bisa langsung ditranslasikan menjadi "produksi"

File: content/id/_index.html [hunk 2]
```diff
@@ -29,7 +29,7 @@
 {{% blocks/feature image="suitcase" %}}
 #### Dapat Dijalankan di Berbagai Platform
 
-Kubernetes sebagai <i>open source</i> memberikan kamu kebebasan untuk menggunakan <i>on-premise</i>, <i>hybrid</i>, atau <i>public cloud infrastructure</i>, memberikan kamu kemudahan untuk memindahkan workloads yang kamu miliki.
+Kubernetes sebagai <i>open source</i> memberikan kamu kebebasan untuk menggunakan <i>on-premise</i>, <i>hybrid</i>, atau <i>public cloud infrastructure</i>, memberikan kamu kemudahan untuk memindahkan <i>workloads</i> yang kamu miliki.
```
Comment (ariscahyadi) [inline content/id/_index.html:32]:
```suggestion
Kubernetes sebagai <i>open source</i> memberikan kamu kebebasan untuk menggunakan <i>on-premise</i>, <i>hybrid</i>, atau infrastruktur _cloud_ publik, memberikan kamu kemudahan untuk memindahkan <i>workloads</i> yang kamu miliki.
```
Cetak miring bisa pake "_kata_"

File: content/id/docs/setup/_index.md [hunk 3]
```diff
@@ -30,5 +30,5 @@ Jika kamu sedang belajar Kubernetes, gunakan alat yang didukung oleh komunitas K
 
 Pada saat mengevaluasi sebuah solusi untuk lingkungan produksi, pertimbangkan aspek dari pengoperasian klaster Kubernetes (atau _abstractions_) yang ingin kamu kelola atau membiarkan penyedia.
```
Comment (ariscahyadi) [inline content/id/docs/setup/_index.md:31]:
```suggestion
Pada saat mengevaluasi sebuah solusi untuk lingkungan produksi, pertimbangkan aspek dari pengoperasian klaster Kubernetes atau abstraksi yang ingin kamu kelola atau membiarkan penyedia.
```
https://kbbi.kemdikbud.go.id/entri/abstraksi

File: content/id/docs/setup/_index.md [hunk 4]
```diff
@@ -30,5 +30,5 @@ Jika kamu sedang belajar Kubernetes, gunakan alat yang didukung oleh komunitas K
...
 
-[Kubernetes Partners](https://kubernetes.io/partners/#conformance) termasuk dari daftar dari penyedia [Certified Kubernetes](https://github.com/cncf/k8s-conformance/#certified-kubernetes).
+[Kubernetes _Partners_](https://kubernetes.io/partners/#conformance) termasuk dari daftar dari penyedia [_Certified_ Kubernetes](https://github.com/cncf/k8s-conformance/#certified-kubernetes).
```
Comment (ariscahyadi) [inline content/id/docs/setup/_index.md:33]:
```suggestion
[Kubernetes Partner](https://kubernetes.io/partners/#conformance) termasuk dari daftar dari penyedia [_Certified_ Kubernetes](https://github.com/cncf/k8s-conformance/#certified-kubernetes).
```
https://kbbi.kemdikbud.go.id/entri/partner

**Review by ariscahyadi — COMMENTED** [review]
Thanks @rendiputra for your contribution. 
I have several comments and also please check some useful information here https://kubernetes.io/id/docs/contribute/localization_id/

File: content/id/_index.html (hunk 2 above)
Comment (rendiputra) [reply content/id/_index.html]: noted mas

File: content/id/_index.html (hunk 1 above)
Comment (rendiputra) [reply content/id/_index.html]: Siap mas @ariscahyadi

**Review by irvifa — COMMENTED** [review]
/lgtm
/approve

**Review by irvifa — COMMENTED** [review]
/approve

**Review by ariscahyadi — COMMENTED** [review]
Thanks @rendiputra for your contribution.
Please fell free to continue your contribution by taking task in this issue https://github.com/kubernetes/website/issues/28772

**Comment by reylejano** [comment]
/retitle [id] Fixing inconsistency italic word for foreign word

**Review by divya-mohan0209 — COMMENTED** [review]
Taking approval from [this comment](https://github.com/kubernetes/website/pull/31234#pullrequestreview-847405480) by @irvifa
/approve

**Comment by divya-mohan0209** [comment]
[ariscahyadi](https://github.com/ariscahyadi) could you please unhold & signoff if everything looks okay?

**Comment by Sea-n** [comment]
Ping @ariscahyadi, this PR is been placed for a while.

**Comment by Sea-n** [comment]
Hey @rendiputra,

It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?

**Pull request merged**